test/other/embind_tsgen_ignore_2.d.ts

```diff
@@ -101,8 +101,9 @@ interface EmbindModule {
   DerivedClass: {};
   a_bool: boolean;
   an_int: number;
-  optional_test(_0: Foo | undefined): number | undefined;
+  optional_test(_0?: Foo): number | undefined;
```

Comment:
Can the return type here also use `?`?

Reply:
As far as I can tell that's not currently supported. There's a bunch of issues related to this https://github.com/microsoft/TypeScript/issues/51332